src/features/cell-renderers/lastModifiedByCellRenderer.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+import * as React from 'react';
+import { FormattedMessage, FormattedDate, FormattedRelative } from 'react-intl';
```

Comment:
You will have to use the intl method instead of the component here for these renderers. Virtualize table will auto add a tooltip (title) to the cell if it's a string that got passed back. It won't if it's a component.

Reply:
You will also have to make sure FormattedUser returns a string and not a component.